src/chrome/content/rules/arte.xml

```diff
@@ -64,19 +57,25 @@ Fetch error: http://info.arte.tv/ => https://info.arte.tv/: Too many redirects w
 
 	wrong content:
 		- forums.arteradio.com
+
+	Redirect to HTTP:
+		- info.arte.tv
+
+	Too many redirects:
```

Comment:
@koops76 They have different behavior here (one redirects to HTTP other to HTTPS after a while) but i'm ok with what you say